### ISSUE TITLE
Activity records of databases

### ISSUE THREAD

**Issue by temporallyaccount** [keepassxreboot/keepassxc#5465]
#### Summary
Keepassxc has auto save and auto backup features however even something happened mistakenly then we have to spend some time check backup files about what lost or changed. Sometimes we see save buttons appear (non-grayed) then click it to save while don't have clear idea what did for past hour or 5hrs etc.
This's mind blowing thing for someone that have so many entries and multiple databases while spend a busy lifestyle.

Add a feature to display 'activity record'(log) Tools > Activity Record
```
[2020-09-24T14:45:58] INFO: Database "Business" /General/eMail/ <entry title/ID> password modified 
[2020-09-24T14:45:58] INFO: Database "Network" /General/Notes/ <entry title/ID> username modified
[2020-09-24T14:45:58] INFO: Database "Business" /General/eMail/ <entry title/ID> new entry added 
[2020-09-24T14:45:58] INFO: Database "Business" /Network/Drive/ <entry title/ID> deleted
[2020-09-24T14:45:58] INFO: Database "Devices" /Drive/Backup/ <entry title/ID> notes added
``` 

User able to view 'active records' before click save button as well, means it help to review past changes before save.

#### Examples

![Screenshot from 2020-09-24 15-38-19](https://user-images.githubusercontent.com/64679174/94122007-34980800-fe7c-11ea-9939-dc43febe919f.png)

**Comment by droidmonkey** [comment]
Good idea, goes along with our desire to have a transactional saving system (will allow undo/redo)

**Comment by aluyama** [comment]
I have lost several credentials already. please give a priority to implement this feature.

**Comment by droidmonkey** [comment]
We offer entry history already

**Comment by aluyama** [comment]
How to view? :/

**Comment by droidmonkey** [comment]
https://keepassxc.org/docs/KeePassXC_UserGuide.html#_history

**Comment by aluyama** [comment]
Ah! thank you. i didn't see it

**Comment by droidmonkey** [comment]
This is a duplicate request to one that has more amplifying information
